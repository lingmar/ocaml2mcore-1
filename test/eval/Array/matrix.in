let _ =
  let a = Array.make_matrix 3 4 0 in
  a.(1) <- Array.make 4 1;
  a.(2) <- Array.make 4 2;

  print_string "Length is ";
  print_int (Array.length a);
  print_endline "";

  Array.iter print_int a.(0);
  print_endline "";
  Array.iter print_int a.(1);
  print_endline "";
  Array.iter print_int a.(2);

  ()
