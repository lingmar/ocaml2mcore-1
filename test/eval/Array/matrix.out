Length is 3
0000
1111
2222
